### PR TITLE
Balanced Bracket Problem

### PR DESCRIPTION
**Pull request by Prerna-07**
closes : #1683

### CODE REVIEW

**Comment by Prerna-07** [comment]
@Rohan27s pls check once more i have created everything one more

**Review by yasharth291 — APPROVED** [review]

File: Stack/Balanced Bracket Problem.Java [hunk 1]
```diff
@@ -0,0 +1,95 @@
+/* Balanced Bracket Problem
+Explanation :- To check if the expression is balanced i.e. closing brackets and opening brackets match up well.
+
+
+It will be an iterative solution in which loop will iterate over each character of expression, if opening brackets encountered then simply push in stack, if closing brackets 
+encountered then if stack is empty return false or if top element of stack is not brackets significant other return false. After Iterating whole expression if stack not empty 
+means some opening brackets doesn't have closing hence not balanced. At end if stack is empty means expression is balanced.
+
```
Comment (Rohan27s) [inline Stack/Balanced Bracket Problem.Java:8]: Remove unnecessary spaces

File: Tree/readme.md [hunk 2]
```diff
@@ -56,8 +56,8 @@ that, this property should be satisfied at every node in the tree.
 * Lowest common ancestor in a binary tree ----> [Python](/Code/Python/LCA_in_binary_tree.py)
 * Right Side View of Binary Tree ----> [C++](/Code/C++/Right_Side_View_of_Binary_Tree.cpp)
 * Root to Leaf Path with Given Sum ----> [C++](/Code/C++/Root_to_leaf_path_with_given_sum.cpp)
-* Searching in BST ----> [C++](/Code/C++/searching_in_bst.cpp)
```
Comment (Rohan27s) [inline Tree/readme.md:59]: Unnecessary commit you just need to edit Readme of Stack

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Review by Rohan27s — APPROVED** [review]

**Comment by Rohan27s** [comment]
Why have you deleted tree/Readme in your commit. I asked you to remove the commit of the read me file, not to delete the tree/readme file in your commit

**Comment by Prerna-07** [comment]
@Rohan27s sorry i am trying to getting it back

**Comment by Prerna-07** [comment]
@Rohan27s i have restored the readme file. Pls guide me how to remove that commmit

**Comment by Prerna-07** [comment]
@Rohan27s I have resolved the problem pls check.

**Comment by Prerna-07** [comment]
@Rohan27s ok sir

**Comment by Prerna-07** [comment]
@Rohan27s  if i delete my repository will it effect the merged issues?

**Comment by Rohan27s** [comment]
No @Prerna-07

**Comment by Prerna-07** [comment]
@Rohan27s I have done all necessary changes pls check my pull request #1794.

**Comment by Rohan27s** [comment]
@Prerna-07 don't comment on a closed pr. I will check it wait.

**Pull request closed without merging**